package.json

```diff
@@ -14,30 +14,32 @@
     },
     "homepage": "https://github.com/wednesday-solutions/node-js-hapi-template#readme",
     "dependencies": {
-        "esm": "^3.2.25",
         "@hapi/boom": "^7.4.9",
+        "@hapi/catbox-redis": "^5.0.5",
         "@hapi/hapi": "^18.4.0",
+        "@hapi/inert": "5.2.2",
         "@hapi/joi": "^15.1.1",
-        "loadash": "^1.0.0",
+        "@hapi/vision": "^5.5.4",
+        "dotenv": "^8.2.0",
+        "esm": "^3.2.25",
         "hapi-cors": "^1.0.3",
+        "hapi-pagination": "^3.0.0",
         "hapi-rate-limit": "^4.1.0",
+        "hapi-swaggerui": "https://github.com/wednesday-solutions/hapi-swaggerui#master",
+        "loadash": "^1.0.0",
+        "lodash": "^4.17.15",
         "map-keys-deep": "0.0.2",
+        "moment": "^2.24.0",
+        "mysql2": "^1.7.0",
+        "nodemon": "^2.0.4",
         "qs": "^6.9.1",
         "semver": "^6.3.0",
-        "wurst": "^4.1.1",
-        "uuid": "^3.3.3",
-        "moment": "^2.24.0",
         "sequelize": "^5.21.2",
-        "mysql2": "^1.7.0",
-        "hapi-swaggerui": "https://github.com/wednesday-solutions/hapi-swaggerui#master",
-        "@hapi/inert": "5.2.2",
-        "@hapi/vision": "^5.5.4",
-        "hapi-pagination": "^3.0.0",
-        "lodash": "^4.17.15",
-        "@hapi/catbox-redis": "^5.0.5"
+        "uuid": "^3.3.3",
+        "wurst": "^4.1.1"
     },
     "scripts": {
-        "start": "pm2 start ecosystem.config.js --env development",
```

Comment:
we need to add start scripts for dev, so one that sets the ENV variable correctly and starts the server using node instead of nodemon